src/transformers/onnx/__main__.py

```diff
@@ -172,6 +173,62 @@ def main():
 
         validate_model_outputs(onnx_config, preprocessor, model, args.output, onnx_outputs, args.atol)
         logger.info(f"All good, model saved at: {args.output.as_posix()}")
+        logger.info(
```

Comment:
Here I think we should issue a proper deprecation warning with `warnings.warn(xxx, FutureWarning)`

Reply:
Done!